### PR TITLE
perf(travis): Faster builds

### PR DESCRIPTION
**Pull request by adityahase**
Similar to https://github.com/frappe/frappe/pull/7964

https://frappe.erpnext.com/desk#Form/Timesheet/TS-00818

### CODE REVIEW

**Comment by adityahase** [comment]
Thanks to @karthikeyan5

File: erpnext/hr/doctype/staffing_plan/test_staffing_plan.py [hunk 1]
```diff
@@ -92,5 +92,5 @@ def make_company():
 	company.abbr = "_TC10"
 	company.parent_company = "_Test Company"
 	company.default_currency = "INR"
-	company.country = "India"
+	company.country = "Pakistan"
```
Comment (Mangesh-Khairnar) [inline erpnext/hr/doctype/staffing_plan/test_staffing_plan.py:95]: Why is this change needed?

Comment (netchampfaris) [reply erpnext/hr/doctype/staffing_plan/test_staffing_plan.py:95]: @Mangesh-Khairnar 
https://github.com/frappe/erpnext/pull/18488/commits/0f072d7d40ff820ebd237a02b96e81f629661854

**Review by deepeshgarg007 — APPROVED** [review]
